flaskr.py

```diff
@@ -80,4 +80,16 @@ def login():
 def logout():
 	session.pop('logged_in', None)
 	flash('You were logged out')
+	return redirect(url_for('show_entries'))
+
+@app.route('/delete=<int:entry_id>', methods=['GET','POST'])
+def delete_entry(entry_id):
+	error=None
+	if request.method == 'POST':
+		db=get_db()
+		print(entry_id)
+		db.execute('delete from entries where id= (?)', (entry_id,))
```

Comment:
General note: for learning purpose it is OK (and even good) to use DB this way (just execute SQL queries).
But in real application we would use something like SQLAlchemy to perform DB queries.
And for raw SQL I can recommend also this styleguide - http://www.sqlstyle.guide/